web/src/main/java/org/akaza/openclinica/controller/openrosa/QueryServiceImpl.java

```diff
@@ -76,7 +76,9 @@ public void process(QueryServiceHelperBean helperBean, SubmissionContainer conta
         helperBean.setItemOrdinal(itemOrdinal);
         helperBean.setItemNode(itemNode);
         helperBean.setItemData(getItemData(helperBean));
-        // helperBean.setResStatus(resolutionStatusDao.findByResolutionStatusId(1));
+        if (helperBean.getItemData() == null) {
```

Comment:
There's a new pull request for these code changes. Thanks
https://github.com/OpenClinica/OpenClinica/pull/769